src/opendr/perception/object_detection_2d/retinaface/algorithm/dataset/imdb.py

```diff
@@ -0,0 +1,320 @@
+"""
+General image database
```

Comment:
Does this require a copyright mention? You made it or is it part of the deepinsight implementation?

Reply:
This is part of the deepinsight implementation. I'm not sure why it might need a copyright mention, do you think something should be added to the documentation of the method perhaps?

Reply:
Well since it's part of deepinsight I think this [README](https://github.com/tasostefas/opendr_internal/blob/object-detection-2d-auth/src/opendr/perception/object_detection_2d/retinaface/README.md) and the corresponding copyright info should cover for it, so I think it's fine